README.md

```diff
@@ -15,8 +15,25 @@ This repository contains the gamelogic of the game Unvanquished.
 You need to download the game's assets in addition to that to make it run.
 See below for build and launch instructions.
 
+## Workspace requirements
+
+To fetch and build Unvanquished, you'll need:
+`git`,
+`cmake`,
+`python` ≥ 2,
```

Comment:
Python == 2 is required if you want to build NaCl. Otherwise LGTM.

Reply:
is building NaCl required to build Unvanquished? I guess the python requisite is for some code generation unrelated to NaCl, right?

Reply:
To build NaCl VMs. The 2 or 3 requirement is for CBSE (sgame).

Reply:
ah, that's not cool… then it's probably better to just say python2 if NaCl requires it and CBSE can deal with it…